src/main/resources/templates/lemminx/clientSettings.json

```diff
@@ -2,6 +2,9 @@
   "caseSensitive": true,
   "blockCommentPrefix": "<!--",
   "blockCommentSuffix": "-->",
+  "breadcrumbs": {
```

Comment:
Disabled by default for XML and HTML (below). Those languages already have first-class breadcrumbs support in all JetBrains IDEs.